src/domain/entities/Option.ts

```diff
@@ -6,3 +6,5 @@ export type Option = {
     code: string;
     sortOrder: number;
 };
+
+export const DEFAULT_VALID_LENGTH = 230;
```

Comment:
[nothing to do] That's the limit imposed by DHIS2, but, as we are building d2 tools, we can say it's ok to impose it  in the model.